TEACHING.md

```diff
@@ -0,0 +1,85 @@
+
+Saturday: The rest of the [curriculum](README.md#outline). 
+
+Preparation
+-----------
+Print the [student cheatsheet](outline/cheatsheet.md) and hand out to students at the beginning of the day on Saturday.
+
+Room Setup
+----------
+It works best to have attendees sit around tables facing each other. That encourages attendees to help each other and make it easier for TAs to get to them to help out.
+
+Presentation of the Material
+----------------------------
+Most workshops have both teachers and TAs. Teachers present the material at the front of the room, and TAs and teachers help students as they work through the exercises on their own and with their tablemates. This is different from how RailsBridge usually works, where students generally work through the material in groups with the help of TAs.
+
+This is not the only way to present the material. It is just the way we have seen it done at most of the workshops so far. Please feel free to experiment.
+
+Light Table
+-----------
+Make sure to spend some time during the Installfest or at the beginning of Saturday showing a few key things to do in Light Table:
+
+- Open the command pane with `Ctrl-Space`
+- Start an Instarepl in the command pane by typing `Insta`
+- Set the font size or line numbers in `Settings: User behaviors` 
+- Evaluate a file with `Cmd-Shift-Enter` or, on Windows, `Ctrl-Shift-Enter`
+
+Students who want to do more can take a look at the [Light Table tutorial](http://docs.lighttable.com/tutorials/full/)
+
+Beginner and Advanced Tracks
+----------------------------
+You can survey the attendees ahead of time or during the Installfest to find out their level of programming experience. If you have multiple teachers available, you could present the material in separate rooms at a different pace, or you could present separate material. Or you could split attendees up to sit together according to level in one room. 
+
+Exercises
+---------
+Provide plenty of time for exercises. That is where the real fun happens. TAs should sit with attendees or circulate around the tables as they work on exercises.
+
+Capstone App
+------------
+Budget your time so that you have enough time for the final quil project at the end fo the day.
+
+Outline
+-------
+TODO: add tips under each heading.
+
+* [Getting Set Up](outline/setup.md)
+* [Introduction to Programming with Clojure](outline/intro.md)
+* [Data Structures](outline/data_structures.md)
+* [Functions](outline/functions.md)
+* [More Simple Values](outline/simple_values2.md)
+* [More Functions](outline/functions2.md)
+* [More Data Structures](outline/data_structures2.md)
+* [Flow Control](outline/flow_control.md)
+
+* [Capstone App Pt. 1](https://github.com/ClojureBridge/drawing/blob/master/curriculum/first-program.md)  -- [Repository](https://github.com/ClojureBridge/drawing/blob/master/README.md)
+* [Capstone App Pt. 2](https://github.com/ClojureBridge/drawing/blob/master/curriculum/create-something.md)
+
+
+Supporting Materials
```

Comment:
should we add samples of the info sheet we pass out with links to resources? I liked the one you had for Durham.